.github/workflows/tests.yml

```diff
@@ -129,6 +129,12 @@ jobs:
     with:
       testfilter: HIP904
 
+  WHBAR:
+    name: WHBAR Contract Test Suite
+    uses: ./.github/workflows/test-workflow.yml
+    with:
+      testfilter: WHBAR
```

Comment:
I suggest to only execute it of changes on the given test or src folder for WHBAR

Reply:
I don't think we have that kind of mechanism yet.

We have a base flow, which is defined [here](https://github.com/hashgraph/hedera-smart-contracts/blob/main/.github/workflows/test-workflow.yml). And here is [the list](https://github.com/hashgraph/hedera-smart-contracts/blob/main/.github/workflows/tests.yml#L23) of all jobs. They are executed no matter if there is a change or not.

We may open a new task for making the job execution mandatory only if there is a change. The change must be done on a global level, not only for `WHBAR` IMO.